### PR TITLE
Check tar command

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2736/

File: AppController/lib/helperfunctions.rb [hunk 1]
```diff
@@ -478,6 +478,13 @@ def self.app_has_config_file?(tar_gz_location)
     end
   end
 
+
+  # Prepare the application code to be run by AppServers.
+  #
+  # Args:
+  #   app_name: A String containing the application id.
+  # Returns
+  #   A Boolean indicating if the setup was succesful.
```
Comment (menivaitsi) [inline AppController/lib/helperfunctions.rb]: "successful"

File: AppController/lib/helperfunctions.rb [hunk 2]
```diff
@@ -478,6 +478,13 @@ def self.app_has_config_file?(tar_gz_location)
     end
   end
 
+
+  # Prepare the application code to be run by AppServers.
+  #
+  # Args:
+  #   app_name: A String containing the application id.
+  # Returns
+  #   A Boolean indicating if the setup was successful.
```
Comment (cdonati) [inline AppController/lib/helperfunctions.rb]: Returning a boolean as an indicator of success seems strange to me here. I would recommend throwing an exception when there is a failure.

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2789/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2792/ the last run was taken out by the brown out ...

**Pull request merged**